### ISSUE TITLE
when I use DBCP and neo4j-jdbc to connect Neo4j database,I got a problem

### ISSUE THREAD

**Issue by naniya** [neo4j/neo4j-jdbc#101]
>org.apache.commons.dbcp.SQLNestedException: Cannot create JDBC driver of class 'org.neo4j.jdbc.Driver' for connect URL 'jdbc:neo4j:bolt://127.0.0.1:7687/',
>
> Caused by: java.sql.SQLException: No suitable driver
>	at java.sql.DriverManager.getDriver(DriverManager.java:315)
>	at org.apache.commons.dbcp.BasicDataSource.createConnectionFactory(BasicDataSource.java:1437)

```java
dbProperties.load(GraphDataSource.class.getClassLoader()
						.getResourceAsStream(configFile));
				dataSource = BasicDataSourceFactory.createDataSource(dbProperties);
				Connection conn = getDataSource().getConnection();
```

when this code run in java project,there is no problem,but when I use this code run in javaweb project on tomcat,it tell me no suitable driver. 

when I compare the code of neo4j and mysql,I find some differences:

org.neo4j.jdbc.BaseDriver.java:
```java

public abstract class BaseDriver implements Driver
{
  protected static final String JDBC_PREFIX = "jdbc:neo4j:";
  private String DRIVER_PREFIX;

  protected BaseDriver(String prefix)
    throws SQLException
  {
    this.DRIVER_PREFIX = prefix;
    DriverManager.registerDriver(this);
  }
```

com.mysql.jdbc.Driver.java:
```java
public class Driver extends NonRegisteringDriver
  implements java.sql.Driver
{
  public Driver()
    throws SQLException
  {
  }

  static
  {
    try
    {
      DriverManager.registerDriver(new Driver());
    } catch (SQLException E) {
      throw new RuntimeException("Can't register driver!");
    }
  }
}
```

I don't known if this difference made the problem,or some other reasons.How can I solve the problem?

**Comment by jexp** [comment]
Which driver jar did you configure ? The 3.4MB large one from the GH release / maven?

**Comment by inserpio** [comment]
Hi @naniya, we already solved this problem, but forgot to document (sorry).

When running in tomcat you have to:

1) Add the driver `neo4j-jdbc-bolt-3.0.3.jar` into the `lib` folder
2) Create a `setenv.sh` file with `CLASSPATH=$CATALINA_BASE/lib/neo4j-jdbc-bolt-3.0.3.jar` inside

and restart tomcat. `ClassLoader` will finally take care about the driver.

**Comment by naniya** [comment]
@jexp I used neo4j-jdbc-driver-3.0.jar in my project.I have found one way to solve this problem:changed the code 'Class.forName("org.neo4j.jdbc.Driver");'  to "Class.forName("org.neo4j.jdbc.Driver").newInstance()".But this just worked in the project without DBCP,when I used DBCP ,the code 'Class.forName("org.neo4j.jdbc.Driver");' is wrote in DBCP's code,so I coundn't change it.This problem will appear again. @inserpio His way to solve this problem is correct,but I think we can remove the code 'DriverManager.registerDriver(this)' in org.neo4j.jdbc.BaseDriver.java to static block

**Comment by glaggia-larus** [comment]
this is fixed in #104
